core/coreunix/add.go

```diff
@@ -71,26 +71,27 @@ func NewAdder(ctx context.Context, p pin.Pinner, bs bstore.GCBlockstore, ds ipld
 
 // Adder holds the switches passed to the `add` command.
 type Adder struct {
-	ctx        context.Context
-	pinning    pin.Pinner
-	blockstore bstore.GCBlockstore
-	dagService ipld.DAGService
-	Out        chan interface{}
-	Progress   bool
-	Hidden     bool
-	Pin        bool
-	Trickle    bool
-	RawLeaves  bool
-	Silent     bool
-	Wrap       bool
-	NoCopy     bool
-	Chunker    string
-	root       ipld.Node
-	mroot      *mfs.Root
-	unlocker   bstore.Unlocker
-	tempRoot   *cid.Cid
-	Prefix     *cid.Prefix
-	liveNodes  uint64
+	ctx         context.Context
+	pinning     pin.Pinner
+	blockstore  bstore.GCBlockstore
+	dagService  ipld.DAGService
+	Out         chan interface{}
+	Progress    bool
+	Hidden      bool
+	Pin         bool
+	Trickle     bool
+	RawLeaves   bool
+	Silent      bool
+	Wrap        bool
+	NoCopy      bool
+	Chunker     string
+	FileWrapper func(io.Reader) (io.Reader, error)
```

Comment:
Why are all the field listed as modified? The `FileWrapper` is the only new field right? Also, just to be extra annoying, could we add a comment explaining the function the newly added field? I'm a strong believer in the broken windows theory :)

Reply:
That's gofmt. It aligns everything.